### PR TITLE
sign up to sign in CTA review

### PR DESCRIPTION
**Pull request by hugovalente-pm**
This PR is still not updated with the latest version of the "Verify email!" e-mail screenshot. Once that is available I'll update it.
Putting the PR to trigger the review

### CODE REVIEW

File: docs/cloud/get-started.mdx [hunk 1]
```diff
@@ -24,12 +24,12 @@ you used to install Netdata on that node.
 
 ## Begin the onboarding process
 
-Get started by signing up for Netdata Cloud. Read the [sign in/up](/docs/cloud/manage/sign-in) doc for details on the
+Get started by signing in to Netdata Cloud. Read the [sign in](/docs/cloud/manage/sign-in) doc for details on the
```
Comment (manos-saratsis) [inline docs/cloud/get-started.mdx:27]: @hugovalente-pm "Sign in" without "to Netdata Cloud" is preferred.

Comment (hugovalente-pm) [reply docs/cloud/get-started.mdx]: Just pushed a fix for this on this file and on `docs/cloud/manage/sign-in.md`. please review.

**Review by manos-saratsis — APPROVED** [review]

File: docs/cloud/manage/sign-in.md [hunk 2]
```diff
@@ -55,18 +61,18 @@ a public email service, such as Gmail. You may also want to whitelist/allowlist
...
+To sign in with Google or GitHub OAuth, visit [Netdata Cloud](https://app.netdata.cloud/sign-in?cloudRoute=spaces?utm_source=docs&utm_content=sign_in_button_google_github_section) and click the
 **Continue with Google/GitHub** or button. Enter your Google/GitHub username and your password. Complete two-factor
 authentication if you or your organization has it enabled. 
 
 You are then signed in to Netdata Cloud or directed to the new-user onboarding if you have not signed up previously.
 
 ## Reset a password
 
-Netdata Cloud not store passwords and does not support password resets. All of our sign in/sign up methods do not
+Netdata Cloud not store passwords and does not support password resets. All of our sign in methods do not
```
Comment (OdysLam) [inline docs/cloud/manage/sign-in.md:72]:
```suggestion
Netdata Cloud does not store passwords and does not support password resets. All of our sign in methods do not
```

File: docs/cloud/manage/sign-in.md [hunk 3]
```diff
@@ -1,48 +1,54 @@
...
 
-Click the **Sign up**/**Sign in** button in the email to begin using Netdata Cloud.
+Click the **Verify** button in the email to begin using Netdata Cloud.
 
 To use this same Netdata Cloud account on additional devices, request another sign in email, open the email on that
 device, and sign in.
 
+### Don't have a Netdata Cloud account yet?
+
+Relax! Sign in and we will create one for you!
```
Comment (OdysLam) [inline docs/cloud/manage/sign-in.md:37]: Not sure what you mean by this. We should change the tone of this phrase.

**Review by OdysLam — CHANGES_REQUESTED** [review]

File: docs/cloud/manage/sign-in.md (hunk 3 above)
Comment (hugovalente-pm) [reply docs/cloud/manage/sign-in.md:37]: @OdysLam was trying to put this in-line to the tone we have on our app for the sign in screen, please see printscreen below

image

**Review by manos-saratsis — APPROVED** [review]

File: docs/cloud/manage/sign-in.md (hunk 3 above)
Comment (hugovalente-pm) [reply docs/cloud/manage/sign-in.md]: @OdysLam @manos-saratsis pushed a change to this text to review the tone of voice.

**Review by OdysLam — APPROVED** [review]

**Pull request merged**